Saint Pierre and Miquelon/St Martin: Updated records

stepps00 opened the pull request:
Fixes #13.

This PR includes the following updates:

- [x] New dependency records for St Martin and Saint Pierre and Miquelon, replacing existing region/country records
- [x] Deprecated/supersedes work for the non-current records
- [x] PIP work for the new records and descendants

nvkelso commented on data/112/612/872/3/1126128723.geojson:
```diff
@@ -0,0 +1,206 @@
+    "wof:lastmodified":1498164926,
+    "wof:name":"Saint Martin",
+    "wof:parent_id":102191575,
+    "wof:placetype":"country",
```
Please to `dependency`.

nvkelso commented on data/112/612/872/3/1126128723.geojson:
```diff
@@ -0,0 +1,206 @@
+    "lbl:longitude":-63.059398,
+    "mps:latitude":18.082858,
+    "mps:longitude":-63.059398,
+    "mz:hierarchy_label":1,
```
Should have a `mz:is_current` = 1

stepps00 replied:
(taken care of on line below)

nvkelso commented on data/112/612/872/3/1126128723.geojson:
```diff
@@ -0,0 +1,206 @@
+    "name:fre_x_preferred":[
+        "Saint-Martin"
+    ],
+    "name:fre_x_unknown":[
```
Should these be `fra`?

nvkelso commented on data/112/612/872/3/1126128723.geojson:
```diff
@@ -0,0 +1,206 @@
+        "fre"
+    ],
+    "wof:lang_x_spoken":[
+        "fre"
```
`fra`, too

nvkelso commented on data/856/327/37/85632737.geojson:
```diff
@@ -15,6 +15,69 @@
+    "name:eng_x_preferred":[
+        "Saint Martin"
+    ],
+    "name:fre_x_preferred":[
```
`fra`

nvkelso commented on data/856/327/37/85632737.geojson:
```diff
@@ -15,6 +15,69 @@
     "mps:latitude":18.082858,
     "mps:longitude":-63.059398,
     "mz:hierarchy_label":1,
+    "name:chi_x_preferred":[
```
`zho`

nvkelso commented on data/112/612/872/3/1126128723.geojson:
```diff
@@ -0,0 +1,206 @@
+    "mps:latitude":18.082858,
+    "mps:longitude":-63.059398,
+    "mz:hierarchy_label":1,
+    "name:chi_x_preferred":[
```
`zho`

nvkelso commented on data/856/327/37/85632737.geojson:
```diff
@@ -15,6 +15,69 @@
     "mps:latitude":18.082858,
     "mps:longitude":-63.059398,
     "mz:hierarchy_label":1,
```
let's set this hierarchy label to 0 since it's really a same as situation with it's parent.

nvkelso commented on data/856/327/37/85632737.geojson:
```diff
@@ -99,32 +162,45 @@
+    "wof:lastmodified":1498243924,
     "wof:name":"Saint Martin",
-    "wof:parent_id":85633147,
+    "wof:parent_id":1126128723,
```
Seems like really be a placetype_alt of it's parent dependency?

Is there a pattern like this for all the overseas areas of France, any other dependencies?

nvkelso commented on data/856/738/79/85673879.geojson:
```diff
@@ -114,12 +117,14 @@
     "wof:lang":[
         "fre"
```
fra

nvkelso commented:
Please also resolve the 2 conflicts...

stepps00 commented:
Resolved conflicts, addressed reviewer comments, and updated the record(s) for Saint Pierre and Miquelon (see https://github.com/whosonfirst-data/whosonfirst-data/issues/13#issuecomment-337068540 for discussion about Saint Pierre and Miquelon).

stepps00 commented:
Conflicts fixed in 9f054a42d6ca5273b736ffbc10dd68e7e62081a0.

stepps00 commented:
This was approved, but I added a few property updates (PIPd records, removed string vals from hierararchy, updated edtf props, etc), so tagging @nvkelso for a second glance before importing.

nvkelso commented on data/856/738/79/85673879.geojson:
```diff
@@ -28,14 +30,14 @@
         "Saint Martin"
     ],
-    "name:fra_x_preferred":[
+    "name:fre_x_preferred":[
```
Use `fra` to be consistent with the `wof:lang` property below?

nvkelso commented on data/856/327/37/85632737.geojson:
```diff
@@ -19,12 +20,72 @@
+    "mz:is_current":0,
     "mz:max_zoom":10.0,
     "mz:min_zoom":5.0,
+    "name:chi_x_preferred":[
```
`zho`?

stepps00 commented:
Comments addressed in last commit. Merging.
